keg_elements/crypto.py

```diff
@@ -0,0 +1,47 @@
+import os
+
+import cryptography.hazmat.primitives.ciphers as ciphers
+from cryptography.hazmat.backends import default_backend
+
+
+def aes_cipher(key, iv, mode=None):
+    algo = ciphers.algorithms.AES(key)
+    mode = mode or ciphers.modes.CBC(iv)
+    return ciphers.Cipher(algo, mode, backend=default_backend())
+
+
+def aes_encryptor(key, iv, mode=None):
+    return aes_cipher(key, iv, mode=mode).encryptor()
+
+
+def aes_decryptor(key, iv, mode=None):
+    return aes_cipher(key, iv, mode=mode).decryptor()
+
+
+def aes_encrypt_str(data, key):
```

Comment:
Both this and the `decrypt` seem to be `Fernet` without the HMAC. I think it would make sense to just use `Fernet` and avoid Rolling Our Own Crypto™.